Library/Homebrew/formula_installer.rb

```diff
@@ -1513,8 +1558,6 @@ def forbidden_formula_check
 
   private
 
-  attr_predicate :hold_locks?
```

Comment:
IIUC, instead of a private `attr_predicate`, we can just reference the underlying attr…